### DISCUSSION TITLE
Flash session not working when ImplicitFlush is false

### DISCUSSION THREAD

**Discussion by alimo2** [phalcon/cphalcon#16058] [General]
Hi

I'm using flash session with custom css class and template. When i set implicitFlush to false, output method returns `null`. Otherwise it works and flush messages.

**Comment by niden** [comment]
Created an issue to investigate this.

**Comment by niden** [comment]
Resolved in https://github.com/phalcon/cphalcon/pull/16069

Thank you @alimo2
